File: nordeus_challenge_backend_spring_boot/src/main/java/com/mmileticc/nordeuschallenge/model/dto/MonsterDTO.java
package com.mmileticc.nordeuschallenge.model.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class MonsterDTO {

    private Long id;
    private String name;
    private int difficultyRank;
    private int maxHealth;
    private int attack;
    private int defense;
    private int magic;
}

File: Gauntlet of the Hero - GOTH/Assets/_Scripts/Models/Backend/MonsterDto.cs
using Newtonsoft.Json;

namespace GauntletOfTheHero.Backend.Models
{
    [System.Serializable]
    public class MonsterDto
    {
        [JsonProperty("id")]
        public long id;

        [JsonProperty("name")]
        public string name;

        [JsonProperty("difficultyRank")]
        public int difficultyRank;

        [JsonProperty("maxHealth")]
        public int maxHealth;

        [JsonProperty("attack")]
        public int attack;

        [JsonProperty("defense")]
        public int defense;

        [JsonProperty("magic")]
        public int magic;
    }
}